src/ontology/uberon-edit.obo

```diff
@@ -222484,6 +222484,16 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500107
+name: sphenovomerine suture
+def: "Articulation of sphenoid bone with alae of vomer forms a schindylesis type of joint." [DOI: 10.1097/SCS.0000000000002945, PMID: 27483100]
```

Comment:
DOI: and PMID: should not have a space in between (eg PMID:27483100 instead of PMID: 27483100)